dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumResourceScope.kt

```diff
@@ -122,13 +121,7 @@ internal class RumResourceScope(
         val user = RumFeature.userInfoProvider.getUserInfo()
 
         val finalTiming = timing
-        val duration = if (finalTiming == null) {
-            eventTime.nanoTime - startedNanos
-        } else {
-            // ensure we don't get weird graphs
-            // TODO RUMM-692 find out why we can have resource duration < timing.totalDuration()
-            max(eventTime.nanoTime - startedNanos, finalTiming.totalDuration())
-        }
+        val duration = eventTime.nanoTime - startedNanos
```

Comment:
No need for this check anymore, now that the timing is fixed ;)

Reply:
You don't need the anymore the `totalDuration` function in the `RumResourceTiming` class

Reply:
Indeed, I'll remove it